From: "Ricardo B. Marliere" <ricardo@marliere.net>
Subject: const_structs.checkpatch: add device_type
Date: Sun, 18 Feb 2024 16:25:51 -0300

Since commit aed65af1cc2f ("drivers: make device_type const"), the driver
core can properly handle constant struct device_type.  Make sure that new
usages of the struct already enter the tree as const.

Link: https://lkml.kernel.org/r/20240218-device_cleanup-checkpatch-v1-1-8b0b89c4f6b1@marliere.net
Signed-off-by: Ricardo B. Marliere <ricardo@marliere.net>
Reviewed-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Cc: Joe Perches <joe@perches.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 scripts/const_structs.checkpatch |    1 +
 1 file changed, 1 insertion(+)

--- a/scripts/const_structs.checkpatch~const_structscheckpatch-add-device_type
+++ a/scripts/const_structs.checkpatch
@@ -8,6 +8,7 @@ comedi_lrange
 component_ops
 dentry_operations
 dev_pm_ops
+device_type
 dma_map_ops
 driver_info
 drm_connector_funcs
_
